web/src/games/bashni/index.ts

```diff
@@ -0,0 +1,24 @@
+const Thumbnail = require('./media/thumbnail.png');
+import { GameMode } from 'gamesShared/definitions/mode';
+import { IGameDef, IGameStatus, IGameTranslationStatus } from 'gamesShared/definitions/game';
+import translation from './locales/en.json';
+
+export const bashniGameDef: IGameDef = {
+  code: 'bashni',
+  name: translation.name,
+  contributors: ['ProspectPyxis', 'JosefKuchar', 'mateusazis'],
```

Comment:
Why adding other contributors? I think you were the only one who worked on this?

Reply:
I left the other contributors in because as I said, I basically copy and pasted all the code of checkers for this, so most of this code wasn't written by me. This could be removed if you think it's not necessary, though.

Reply:
ah, ok, up to you :)